docs/apache-airflow/tutorial_taskflow_api.rst

```diff
@@ -312,7 +312,7 @@ The reverse can also be done: passing the output of a TaskFlow function as an in
 
 .. code-block:: python
 
-    @task
+    @task(retries=3)
```

Comment:
I added `retries` the next example. Didn't seem relevant to set a retry on a task that just deserializes an output. Seemed more likely to have retries creating a queue though.

Reply:
Nice idea.